src/Drivers/Imagick/ImagickDriver.php

```diff
@@ -470,12 +470,14 @@ public function flip(FlipDirection $flip): static
 
     public function pixelate(int $pixelate = 50): static
     {
-        $width = $this->getWidth();
-        $height = $this->getHeight();
+        if ($pixelate !== 0) {
```

Comment:
Let's check if `$pixelate` is `0` and then to an early return. This way, we'll lose a level of indentation.

Reply:
ok new commit... hope i did not mess up.
(first contribution ever:x)